src/app/loadout/spreadsheets.ts

```diff
@@ -0,0 +1,158 @@
+      );
+      const stats: CsvRow = {};
+      const equippedArmor = resolvedLoadout.resolvedLoadoutItems.filter(
+        (i) => i.loadoutItem.equip && i.item.bucket.inArmor,
+      );
+      if (equippedArmor.length === 5) {
+        const calculatedStats = getLoadoutStats(
+          defs,
+          loadout.classType,
+          subclass,
+          equippedArmor.map((i) => i.item),
+          resolvedLoadout.resolvedMods.map((mod) => mod.resolvedMod),
+          includesFontStats,
+          inGameArmorEnergyRules,
+        );
+        for (const [statHash_, val] of Object.entries(calculatedStats)) {
+          const def = defs.Stat.get(parseInt(statHash_, 10));
+          if (def) {
+            stats[def.displayProperties.name] = val.value;
+          }
+        }
+      }
+
+      const className =
+        loadout.classType === DestinyClass.Unknown
+          ? t('Loadouts.Any')
+          : Object.values(defs.Class.getAll()).find((c) => c.classType === loadout.classType)!
+              .displayProperties.name;
+
+      const localizeResolvedPlugs = (list: { plug: PluggableInventoryItemDefinition }[]) =>
+        list.map((mod) => mod.plug.displayProperties.name);
+
+      const sortItems = (category: D2BucketCategory) =>
+        compareBy((item: ResolvedLoadoutItem) =>
+          buckets.byCategory[category].findIndex((b) => b.hash === item.item.bucket.hash),
+        );
+
+      const equippedItems = resolvedLoadout.resolvedLoadoutItems
+        .filter((i) => i.loadoutItem.equip && i.item.bucket.hash !== BucketHashes.Subclass)
+        .sort(sortItems('Weapons'));
+      const unequippedItems = resolvedLoadout.resolvedLoadoutItems
+        .filter((i) => !i.loadoutItem.equip)
+        .sort(sortItems('Armor'));
+
+      return {
+        Id: loadout.id,
+        'Class Type': className,
+        Name: loadout.name,
+        Notes: loadout.notes,
+        'Last Edited': loadout.lastUpdatedAt
+          ? new Date(loadout.lastUpdatedAt).toDateString()
+          : undefined,
+        ...stats,
+        Subclass: subclass?.item.name,
+        Abilities: localizeResolvedPlugs(abilities),
+        Aspects: localizeResolvedPlugs(aspects),
+        Fragments: localizeResolvedPlugs(fragments),
+        'Equipped Items': equippedItems.map((i) => i.item.name),
+        'Unequipped Items': unequippedItems.map((i) => i.item.name),
+        Mods: resolvedLoadout.resolvedMods.map((mod) => mod.resolvedMod.displayProperties.name),
+        'Artifact Season': loadout.parameters?.artifactUnlocks?.seasonNumber,
+        'Artifact Unlocks': loadout.parameters?.artifactUnlocks?.unlockedItemHashes.map(
+          (modHash) => defs.InventoryItem.get(modHash)?.displayProperties.name,
+        ),
+        'Exotic Armor':
+          loadout.parameters?.exoticArmorHash !== undefined
+            ? getLockedExotic(defs, loadout.parameters?.exoticArmorHash)?.[1]
+            : undefined,
+      };
```

Comment:
Not sure if you'd want to add anything more from loadout parameters - or even just whether there are LO settings or not.